### PR TITLE
updating caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @a-lai2
- [ ] @vvaths
- [ ] @CarmenFerretti
- [ ] @Jakubw1
- [ ] @abarlowSOD
- [ ] @awojna
- [ ] @maperkowski
- [ ] @marcinhk
- [ ] @massetm
- [ ] @przempazio
- [ ] @pstq
- [ ] @ramosalvin813
- [ ] @randyb77
- [ ] @vyas000

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/03-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/03-vvaths.md:5]: looks good!

File: _slides/03-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-Caption here
+amazing new caption!
```
Comment (vyas000) [inline _slides/03-vvaths.md:6]:
```suggestion
Magic Amazing New Caption!!!
```

**Review by vyas000 — CHANGES_REQUESTED** [review]
good job! please make changes requested!

File: _slides/03-vvaths.md (hunk 1 above)
Comment (vvaths) [reply _slides/03-vvaths.md:5]: thanks!

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**